UserService/app/src/main/java/com/bookmyshow/userservice/AppConfig.java

```diff
@@ -0,0 +1,15 @@
+package com.bookmyshow.userservice;
+
+import org.modelmapper.ModelMapper;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class AppConfig {
+
+    @Bean
+    public ModelMapper modelMapper(){
+        return new ModelMapper();
+    }
```

Comment:
Rename the ApplicationMain class as UserServiceMain to adhere naming conventions